feg/gateway/services/session_proxy/credit_control/gy/config.go

```diff
@@ -207,20 +210,23 @@ func GetGyGlobalConfig() *GyGlobalConfig {
 	configsPtr := &mconfig.SessionProxyConfig{}
 	err := managed_configs.GetServiceConfigs(credit_control.SessionProxyServiceName, configsPtr)
 	siStr := diameter.GetValueOrEnv(OCSServiceIdentifierFlag, OCSServiceIdentifierEnv, "")
+	avp437 := diameter.GetValueOrEnv(OCSServiceIdentifierFlag, DisableRequestedGrantedUnitsAVPEnv, "")
```

Comment:
copy/paste, should be:
avp437 := diameter.GetValueOrEnv(DisableRequestedGrantedUnitsAVPFlag, DisableRequestedGrantedUnitsAVPEnv, "")